File: C/Ex28.c
#include<stdio.h>
#include<stdlib.h>
#include<conio.h>
#include<ctype.h>

//Função principal do programa
int main(int argc, char const *argv[])
{
// Declaração das variáveis
int res, num;

// PROGRAMA 28 - EXIBIR A TABUADA DO 1 AO 20, NO INTERVALO DE 1 A 10
                
do
{
system("cls");   
//Exibe na tela 
printf("EXIBIR A TABUADA DO 1 AO 20 NO INTERVALO DE 1 A 10");
printf("\nDigite um numero do intervalo:");
scanf("%i",&num);
while(num<0 || num>10){
    printf("Erro!\nApenas valores do intervalo. Digite novamente: ");
    scanf("%i", &num);
}

for (int i=0; i<=20; i++)
{  
printf("%i x %i = %i\n", num, i, i*num);
}
printf("\nDigite 1 para continuar ou 2 para sair: ");
scanf("%i", &res);
}while(res !=2);

return 0;
}

File: PYTHON/Ex28.py
''' Exibir a tabuada dos valores de um a vinte, no intervalo de um a dez.
Entre as tabuadas, solicitar que o usuário pressione uma tecla'''

print("Exibir a tabuada do 1 ao 20 no intervalo do 1 ao 10")

num = int(input('Digite um número do intervalo: '))

while num <=0 or num>10:
    print(f"Erro! Apenas valores do intervalo")
    num = int(input("Digite novamente: "))

for i in range(1,21,1): #range ( parte do 1, vai até inteiro menor que 21, no passo 1 em 1 )  
    res = num * i
    print(f'{num} x {i} = {res}')
  

